src/components/GlobalPreferences/HelpAndFeedback/HelpAndFeedback.js

```diff
@@ -110,8 +110,8 @@ function HelpAndFeedback({ historyPush, locator, onClose }) {
             </span>
           </label>
           <Info>
-            The embedded console lets you interact with your DAO as you would
-            with Aragon CLI, but inside the client. Be noted that this feature
+            The embedded console lets you interact with your organization as you would
+            with aragonCLI, but inside the frontend client. Please note that this feature
```

Comment:
Shouldn’t we call it “Aragon client” rather than “frontend client”?

Reply:
@bpierre normally, yes, but here, because aragonCLI is also mentioned in the same sentence, I wanted to explicitly distinguish the "frontend client" (because aragonCLI is a command-line interface for Aragon, and the "Aragon client" is a frontend or "graphical" interface for Aragon).

At least -- that was the logic behind that wording! Open to changing it if you think something else (such as "Aragon client") would get the point across more clearly :)

Reply:
Thanks for explaining the reasoning behind this!

> Open to changing it if you think something else (such as "Aragon client") would get the point across more clearly :)

No strong opinion, I was just wondering if it was a mistake. Let’s keep yours :+1:

Reply:
Ah, I was going to say, by consistently using "the Aragon client" I think users will be able to make the difference.

But in any case, I think we could also reword this section in the future to remove mentioning the CLI since they're two completely different feature sets and have different interaction patterns.